### DISCUSSION TITLE
Windows｜Linux中如何重启开启停止服务？

### DISCUSSION THREAD

**Discussion by lonely06** [jeessy2/ddns-go#881] [Q&A] [answered]
如题。。。。
另，使用`./ddns-go -s install`命令在Windows｜Linux中进行安装时，安装的文件是在哪里？能否可以自定义？

**Comment by jeessy2** [comment]
暂不能重启，可以PR
安装时，文件在当前运行的目录

**Reply by lonely06** [reply]
了解了，有时间研究一下
好的

**Reply by jeessy2** [reply, marked as answer]
v5.6.4支持了 `./ddns-go -s restart`
